Sources/BraintreeSEPADirectDebit/BTSEPADirectDebitClient.swift

```diff
@@ -152,20 +152,15 @@ import BraintreeCore
     ) {
         returnedToAppAfterPermissionAlert = false
         
-        self.webAuthenticationSession.start(
-            url: url,
-            context: context,
-            sessionDidDisplay: { [weak self] didDisplay in
-                if didDisplay {
-                    self?.apiClient.sendAnalyticsEvent(BTSEPADirectAnalytics.challengePresentationSucceeded)
-                } else {
-                    self?.apiClient.sendAnalyticsEvent(BTSEPADirectAnalytics.challengePresentationFailed)
-                }
-            },
-            sessionDidComplete: { url, error in
-                self.handleWebAuthenticationSessionResult(url: url, error: error, completion: completion)
+        self.webAuthenticationSession.start(url: url, context: context) { [weak self] didDisplay in
```

Comment:
I tend to prefer the multiple trailing close syntax since that's what Apple preferred/moved to in [this proposal](https://github.com/apple/swift-evolution/blob/main/proposals/0279-multiple-trailing-closures.md). But if you feel strongly about it we can revert.

Reply:
Nice find!

> Informally, the new syntax rules are: 1) The first trailing closure drops its argument label (like today). 2) Subsequent trailing closures require argument labels. These rules seem to work well in practice, because functions with multiple closure arguments tend to have one that is more primary. Often any additional closure arguments are optional.

Maybe then we would need to update our "primary" closure, which is `sessionDidComplete` to be the first one.

Reply:
I guess just since `didComplete` and `didDisplay` are not super easy to distinguish, I like the explicit naming. Maybe our `didComplete` naming could be improved?

Reply:
We updated this here: https://github.com/braintree/braintree_ios/pull/1044/commits/77298a52d08e6deadb137de795fd8efc4f525f75